frontend/src/plugins/tenantManagement/views/tenantView.js

```diff
@@ -0,0 +1,190 @@
+define(function (require) {
+  var OriginView = require('coreJS/app/views/originView');
+  var Origin = require('coreJS/app/origin');
+
+  var TenantView = OriginView.extend({
+    tagName: 'div',
+    className: function () {
+      var className = 'tenant-item tb-row' + ' ' + this.model.get('_id');
+      return className;
+    },
+    isSelected: false,
+
+    events: {
+      'click': 'onClicked',
+      'click a.edit': 'onEditClicked',
+      'click a.save': 'onSaveClicked',
+      'click a.cancel': 'onCancelClicked',
+
+      'click button.disableTenant': 'onDisableTenantClicked',
+      'click button.enableTenant': 'onEnableClicked'
+    },
+
+    preRender: function () {
+      this.listenTo(Origin, 'tenantManagement:tenant:reset', this.resetView);
+      this.listenTo(this.model, 'destroy', this.remove);
+      this.listenTo(this, 'remove', this.remove);
+    },
+
+    render: function () {
+      OriginView.prototype.render.apply(this, arguments);
+      this.applyStyles();
+    },
+
+    applyStyles: function () {
+      if (this.model.get('_isDeleted') === true) {
```

Comment:
We could maybe rewrite this using [$.toggleClass](http://api.jquery.com/toggleclass/#toggleClass-className-state)